content/influxdb/v2.1/reference/internals/metrics.md

```diff
@@ -0,0 +1,1262 @@
+
+### Token service call total
+
+Number of calls.
+
+#### Example
+
+```sh
+# HELP service_token_call_total Number of calls
+# TYPE service_token_call_total counter
+service_token_call_total{method="delete_authorization"} 3
+--
+```
+
+### Token service duration
+
+Duration of calls.
+
+#### Example
+
+```sh
+# HELP service_token_duration Duration of calls
+# TYPE service_token_duration histogram
+service_token_duration_bucket{method="delete_authorization",le="0.005"} 1
+--
+```
+
+### Token service error total
+
+Number of errors encountered.
+
+#### Example
+
+```sh
+# HELP service_token_error_total Number of errors encountered
+# TYPE service_token_error_total counter
+service_token_error_total{code="not found",method="delete_authorization"} 1
+```
+
+urm new call total
+Number of calls.
+
+#### Example
+
+```sh
+# HELP service_urm_new_call_total Number of calls
+# TYPE service_urm_new_call_total counter
+service_urm_new_call_total{method="find_urms"} 6451
+```
+
+urm new duration
+Duration of calls.
+
+#### Example
+
+```sh
+# HELP service_urm_new_duration Duration of calls
+# TYPE service_urm_new_duration histogram
+service_urm_new_duration_bucket{method="find_urms",le="0.005"} 6198
+--
+```
+
+user new call total
+Number of calls.
+
+#### Example
+
+```sh
+# HELP service_user_new_call_total Number of calls
+# TYPE service_user_new_call_total counter
+service_user_new_call_total{method="find_permission_for_user"} 4806
+--
+```
+
+user new duration
+Duration of calls.
+
+#### Example
+
+```sh
+# HELP service_user_new_duration Duration of calls
+# TYPE service_user_new_duration histogram
+service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4039
+--
+```
+
+## InfluxDB task statistics
+
+### Task executor errors counter
+
+The number of errors thrown by the executor with the type of error (ex. Invalid, Internal, etc.)
+
+#### Example
+
+```sh
+# HELP task_executor_errors_counter The number of errors thrown by the executor with the type of error (ex. Invalid, Internal, etc.)
+# TYPE task_executor_errors_counter counter
+task_executor_errors_counter{errorType="internal error",task_type="system"} 1183
+--
+```
+
+### Task executor promise queue usage
+
+Percent of the promise queue that is currently full.
+
+#### Example
+
+```sh
+# HELP task_executor_promise_queue_usage Percent of the promise queue that is currently full
+# TYPE task_executor_promise_queue_usage gauge
+task_executor_promise_queue_usage 0
+```
+
+### Task executor run duration
+
+The duration in seconds between a run starting and finishing.
+
+#### Example
+
+```sh
+# HELP task_executor_run_duration The duration in seconds between a run starting and finishing.
+# TYPE task_executor_run_duration summary
+task_executor_run_duration{taskID="08017725990f6000",task_type="",quantile="0.5"} NaN
+--
+```
+
+### Task executor run latency seconds
+
+Records the latency between the time the run was due to run and the time the task started execution, by task type.
```

Comment:
```suggestion
Records the latency between the time the run was due to run and the time the task started execution, by task type.
```
Latency between the task run's scheduled start time and the execution time, by task type.